### PR TITLE
Fix java 1.6.0.211 compativility

### PR DESCRIPTION
**Pull request by a1b4**
- Change tag `id` to `Id` in the xml body.
  - The server with this Java version return the error `com.sun.org.apache.xml.internal.security.utils.resolver.ResourceResolverException: Can not resolve element with ID Body` with the current implementation. The error is based on the body ID tag, more specifically the XML validation requires that the ID tag must be sent with the initial capitalized

- Change SignedXml to allow the `Id` tag
  - Is unable to sign element by Id attribute when there's a namespace prefix, The xmlCheck return the `Malformed Reference Element'` error the solution is override the `GetIdElement` method.

### CODE REVIEW

**Comment by RodrigoAyala** [comment]
@a1b4 Hi!, Github says that the branch is out of date with `master`. Could you sync the branch please? :)

**Comment by a1b4** [comment]
> @a1b4 Hi!, Github says that the branch is out of date with `master`. Could you sync the branch please? :)

sure, It's done :)

File: Transbank/Webpay/Security/XmlSignature.cs [hunk 1]
```diff
@@ -12,16 +12,20 @@ public XmlSignature(XmlElement element)
...
+        }
+
         public override XmlElement GetIdElement(XmlDocument document, string idValue)
         {
             XmlElement idElem = base.GetIdElement(document, idValue);
 
             if (idElem == null)
             {
                 XmlNamespaceManager nsmanager = new XmlNamespaceManager(document.NameTable);
-                nsmanager.AddNamespace(Constants.WSU, Constants.WSSECURITY_UTILITY);
```
Comment (leosoto) [inline Transbank/Webpay/Security/XmlSignature.cs]: Maybe those two constants are now unused and can be removed?

File: Transbank/Webpay/Security/XmlSignature.cs [hunk 2]
```diff
@@ -12,16 +12,20 @@ public XmlSignature(XmlElement element)
...
+                nsmanager.AddNamespace("wsu", "http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd");
 
-                idElem = (XmlElement)document.SelectSingleNode(String.Format(Constants.WSU_ID, idValue), nsmanager);
```
Comment (leosoto) [inline Transbank/Webpay/Security/XmlSignature.cs]: Same here, maybe Constants.WSU_ID is not used anymore?

**Pull request merged**